src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v2/SwitchControllerV2.java

```diff
@@ -159,4 +160,13 @@
                                                       @RequestBody SwitchPatchDto dto) {
         return switchService.patchSwitch(switchId, dto);
     }
+
+    /**
+     * Return active switch connections to the speakers.
+     */
+    @ApiOperation(value = "Get active switch connections", response = SwitchConnectionsResponse.class)
+    @GetMapping(path = "/{switch_id}}/connections")
```

Comment:
`}}` -> `}`

Reply:
hm... I have fixed the error, at least I remeber it.